atecc608a/main.c

```diff
@@ -223,10 +223,11 @@ psa_status_t test_psa_import_verify(psa_key_handle_t *private_handle,
                                              sizeof(hash), signature,
                                              signature_length));
 
-    psa_close_key(*private_handle);
-    psa_close_key(verify_handle);
     printf("test_psa_import_verify succesful!\n");
+
 exit:
+    psa_close_key(*private_handle);
```

Comment:
`*private_handle` is guaranteed to be initialized because nothing in this function can fail before the call to `psa_open_key` that uses it. But this is very fragile. Please initialize `*private_handle` to 0 before doing anything that can fail.